bigframes/core/indexes/base.py

```diff
@@ -490,17 +490,28 @@ def __getitem__(self, key: int) -> typing.Any:
         else:
             raise NotImplementedError(f"Index key not supported {key}")
 
-    def to_pandas(self) -> pandas.Index:
+    def to_pandas(self, allow_large_results: Optional[bool] = None) -> pandas.Index:
```

Comment:
It's less necessary in this context, since we aren't trying to mimic pandas, but I'd still like to avoid using this parameter positionally.

```suggestion
    def to_pandas(self, *, allow_large_results: Optional[bool] = None) -> pandas.Index:
```

Reply:
Updated.